### PR TITLE
BLIINK Bid Adapter : Add new adapter

### PR DESCRIPTION
**Pull request by Kola-Kola**
## Type of change

- [ ] Bugfix
- [ ] Feature
- [x] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change

Added BLIINK Bidder Adapter

- test parameters for validating bids
```
{
  bidder: 'bliink',
  params: {
    tagId: 32,
    placement: 'video'
  }
}
```

Be sure to test the integration with your adserver using the [Hello World](/integrationExamples/gpt/hello_world.html) sample page.

- contact email of the adapter’s maintainer
- [x] official adapter submission

For any changes that affect user-facing APIs or example code documented on http://prebid.org, please provide:

- A link to a PR on the docs repo at https://github.com/prebid/prebid.github.io/

### CODE REVIEW

**Comment by Kola-Kola** [comment]
> @Kola-Kola can you please add a test file and test coverage for this new adapter?
It is push, thanks.  @ChrisHuie

**Comment by ChrisHuie** [comment]
Docs -> https://github.com/prebid/prebid.github.io/pull/3198

**Comment by samous** [comment]
Hi @gwhigs,

Thank you for your review, we published the changes according to your comments.

BR

File: modules/bliinkBidAdapter.js [hunk 1]
```diff
@@ -57,51 +57,26 @@ export function getOneMetaValue(query) {
...
-
 export function getMetaValue(name, type = 1) {
   const metaList = getMetaList(name)
-
-  let metaContent = ''
-
-  metaList.forEach((meta) => {
-    const method = type === 1 ? getOneMetaValue : getMultipleMetaValue
-    const metaValue = method(`meta[${meta.key}=${meta.value}]`)
+  const value = metaList.some((meta) => {
```
Comment (gwhigs) [inline modules/bliinkBidAdapter.js:62]: `Array.some` [returns a boolean](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some#return_value), not the value.

Probably best to just use a simple for-loop:
```
for (let i=0; i < metaList.length; i++) {
  const meta = metaList[i];
  const metaValue = getOneMetaValue(`meta[${meta.key}=${meta.value}]`);
  if (metaValue) {
    return metaValue
  }
}
return ''
```

Comment (samous) [reply modules/bliinkBidAdapter.js]: Thank @gwhigs for pointing  it out, we pushed the fix

**Review by gwhigs — APPROVED** [review]

**Pull request merged**